### PR TITLE
Completed Html relative links Jimmy Tran 

### PR DESCRIPTION
**Pull request by JimmyTran1206**
Animation

### CODE REVIEW

File: html-relative-links/about-me/about-me.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  </head>
+  <body>
+    <ul>
+      <li><a href="../index.html">Home Page</a></li>
+      <li><a href="about-me.html">About Me</a></li>
+      <li><a href="../projects/projects.html">Projects</a></li>
+      <li><a href="../contact-us/contact-us.html">Contact Us</a></li>
+    </ul>
+    <p>Hello About Me!</p>
+    <img src="../images/c-sharp.png" alt="Girl holding programming book" />
```
Comment (shawnkost) [inline html-relative-links/about-me/about-me.html:19]: You're using the wrong image here! Compare your example to the example gif

File: html-relative-links/contact-us/contact-us.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  </head>
+  <body>
+    <ul>
+      <li><a href="../index.html">Home Page</a></li>
+      <li><a href="../about-me/about-me.html">About Me</a></li>
+      <li><a href="../projects/projects.html">Projects</a></li>
+      <li><a href="contact-us.html">Contact Us</a></li>
+    </ul>
+    <p>Hello Contact Us!</p>
+    <img src="../images/c.png" alt="Girl holding programming book" />
```
Comment (shawnkost) [inline html-relative-links/contact-us/contact-us.html:19]: Using the wrong image here as well!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Jimmy! You seemed to have accidentally added `html-links` as well. You want to avoid adding previous exercises, or you will start running into merge conflicts.

Need to fix a couple images, and then this will be good to go. Make sure you always compare your output to our example output in the instructions

File: html-relative-links/about-me/about-me.html (hunk 1 above)
Comment (JimmyTran1206) [reply html-relative-links/about-me/about-me.html]: thank you Shawn. I have corrected the image links and re-pushed.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**